airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/schemas/common.py

```diff
@@ -36,6 +36,9 @@ def schema_extra(cls, schema: Dict[str, Any], model: Type["BaseModel"]) -> None:
                 if "type" in prop:
                     if allow_none:
                         prop["type"] = ["null", prop["type"]]
+                    if prop["type"] == "array" and prop["items"]:
```

Comment:
let's add unit tests to check this functionality.

Reply:
done